libretroshare/src/pqi/p3netmgr.cc

```diff
@@ -1661,15 +1701,32 @@ void  p3NetMgrIMPL::getIPServersList(std::list<std::string>& ip_servers)
 	mExtAddrFinder->getIPServersList(ip_servers);
 }
 
+void  p3NetMgrIMPL::getCurrentExtIPList(std::list<std::string>& ip_list)
```

Comment:
Do the caller needs barebones IPs as strings?
If not better passing the result of `sockaddr_storage_tostring` which is more generic, contains also the port and it easier to parse.

Reply:
No more info needed, only IP v4 and v6 found by mExtAddrFinder to update DNS server list tooltip in ServerPage.